contentcuration/contentcuration/static/js/edit_channel/channel_set/views/ChannelSetModal.vue

```diff
@@ -0,0 +1,99 @@
+<template>
+
+  <div ref="topmodal" class="modal fade" tabindex="-1">
+    <div class="modal-dialog">
+      <div class="modal-content modal-dialog-default" id="channel_set_modal_content">
+        <div class="modal-header">
+          <h4 class="modal-title">{{ modalTitle }}</h4>
+        </div>
+        <div class="modal-body">
+          <ChannelSetDialog>
+            <component :is="pageType" />
+          </ChannelSetDialog>
+        </div>
+      </div>
+    </div>
+  </div>
+
+</template>
+
+
+<script>
+
+import { mapGetters, mapActions } from 'vuex';
+import { PageTypes } from '../constants';
+import ChannelSetDialog from './ChannelSetDialog.vue';
+import ChannelSetList from './ChannelSetList.vue';
+import ChannelSelectView from './ChannelSelectView.vue';
+
+const pageNameToComponentMap = {
+  [PageTypes.SELECT_CHANNELS]: 'ChannelSelectView',
```

Comment:
Can directly reference the component definition here also, rather than the registered name of the component below. Means that a linter is more likely to warn of undefined variables if you type this name wrong.